content/questions/advanced_security/question-045.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "Which of the following tools can be used for code analysis in a third-party CI system to upload results to GitHub?"
```

Comment:
I feel like you meant to ask which tool can help upload the results of code analysis, but the question is unclear to me and more so asks "which tool can be used for code analysis"

```suggestion
question: "Which tool can be used in a third-party CI system to upload code analysis results to GitHub?"
```